src/unity/toolkits/evaluation/evaluation_interface-inl.hpp

```diff
@@ -1430,7 +1432,7 @@ class flexible_accuracy : public precision_recall_base {
       // All scores.
       case average_type_enum::NONE:
       {
-        return to_variant(accuracy_scores);
```

Comment:
Can we add a small unit test to make sure this stays fixed :)

Reply:
Absolutely. And I'll also update the outputs in the User Guide, where @hoytak first spotted this issue.